packages/mobile/README.md

```diff
@@ -2,14 +2,25 @@
 
 The Audius React Native mobile client
 
+## Requirements
+We recommend using [homebrew](https://brew.sh/) and [pyenv](https://github.com/pyenv/pyenv) to manage dependencies.
+```
+brew install node nvm pyenv ruby cocoapods
+
+pyenv install 3.10
```

Comment:
So is the preinstall script not clear with this? Maybe we can add all this stuff to the root readme?

Reply:
Yeah for sure, maybe we don't specify the versions though and let the config files determine it? When you are in the directory just need to run nvm install and nvm use and it figures it out. Same goes for pyenv and rbenv I think too